news/13540.bugfix.rst

```diff
@@ -0,0 +1,3 @@
+Avoid concurrency issues and improve performance when storing wheels into the
+cache of locally built wheels, when the temporary build directory is on a
+different filesystem than the cache directory.
```

Comment:
I would like it to be just a bit more clear that a temporary directory is created inside the cache directory, e.g.

> Avoid concurrency issues and improve performance, caused by the temporary build directory being on a different filesystem than the cache directory, by building wheels in a temporary sub-directory inside the cache directory.